.github/workflows/rerun-workflows.yml

```diff
@@ -50,6 +50,35 @@ jobs:
               const { WebClient } = require('@slack/web-api');
               const slack = new WebClient('${{ secrets.SLACK_BOT_TOKEN }}');
 
+              // find out which jobs failed
+              const { owner, repo } = context.repo;
+
+              const jobInfo = await github.rest.actions.listJobsForWorkflowRun({
+                owner,
+                repo,
+                run_id: ${{ github.event.workflow_run.id }},
+                per_page: 100,
+              });
+
+              const failedJobs = jobInfo.data.jobs
+                .filter(job => job.status === 'completed' && job.conclusion === 'failure');
```

Comment:
Nit: consistency.

```suggestion
                .filter(job => job.status === "completed" && job.conclusion === "failure");
```

Reply:
come join @kamilmielnik and I on team single-quote